### PR TITLE
update v9 docsite navigation to avoid unnecessary stories

### PR DESCRIPTION
**Pull request by micahgodbolt**
Removed the duplicate theme designer and now pulling directly from the package
Moved the plugin story to non index file, and updated the local storybook to import it
Updated the navigation to move introduction to the top level and ensure it is first selected
adjusted order of folders after components (compat vs preview is up for debate)

**before:**
image

**after:**
image

### CODE REVIEW

File: packages/react-components/react-storybook-addon/.storybook/main.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const rootMain = require('../../../../.storybook/main');
 
 module.exports = /** @type {Omit<import('../../../../.storybook/main'), 'typescript'|'babel'>} */ ({
   ...rootMain,
-  stories: [...rootMain.stories, '../stories/**/*.stories.mdx', '../stories/**/index.stories.@(ts|tsx)'],
+  stories: [...rootMain.stories, '../stories/**/*.stories.mdx', '../stories/**/*.stories.@(ts|tsx)'],
```
Comment (micahgodbolt) [inline packages/react-components/react-storybook-addon/.storybook/main.js:5]: the story for this addon doesn't need to be included in our docs, so switching this package to import non index stories, and changed the filename of the local story

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: apps/public-docsite-v9/src/Concepts/Introduction.stories.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Meta } from '@storybook/addon-docs';
 import pkg from '@fluentui/react-components/package.json';
 import { WhatsNewBoxes } from './components/WhatsNewBoxes';
 
-<Meta title="Concepts/Introduction" />
+<Meta title="Introduction" />
```
Comment (ling1726) [inline apps/public-docsite-v9/src/Concepts/Introduction.stories.mdx:5]: Is this change necessary?

**Review by khmakoto — APPROVED** [review]

**Review by tudorpopams — APPROVED** [review]

**Comment by layershifter** [comment]
This have been already fixed, closing.
